### PR TITLE
astro serviceaccount create now fails with Role required

### PR DESCRIPTION
**Pull request by andriisoldatenko**
fix https://github.com/astronomer/astro-cli/issues/278

### CODE REVIEW

File: cmd/serviceaccount.go [hunk 1]
```diff
@@ -102,6 +103,15 @@ func getValidEntity() (string, string, error) {
 	return entityType, id, nil
 }
 
+func getValidRole() (string, error) {
+	var fullRole string
+	if len(workspaceId) > 0 {
+
+		fullRole = "WORKSPACE_ADMIN"
```
Comment (schnie) [inline cmd/serviceaccount.go]: @andriisoldatenko to match Orbit, I think we want to use `WORKSPACE_VIEWER` as the default role, if the entity type is `WORKSPACE` and `DEPLOYMENT_VIEWER` if the entity type is `DEPLOYMENT`.

Comment (andriisoldatenko) [reply cmd/serviceaccount.go]: @schnie 
Ashb suggested to have flag `--role=admin` and generate `WORKSPACE_ADMIN `
if `--role=viewer`  generate WORKSPACE_VIEWER? right?

Comment (schnie) [reply cmd/serviceaccount.go]: That works for me

Comment (schnie) [reply cmd/serviceaccount.go]: So we should essentially concatenate `entityType` and `role`?

Comment (andriisoldatenko) [reply cmd/serviceaccount.go]: yes

Comment (andriisoldatenko) [reply cmd/serviceaccount.go]: if user skip flag role, should it be viewer as default value?

**Comment by andriisoldatenko** [comment]
practical QA:
```
astro sa cr -d ck05r3bor07h40d02y2hw4n4v --label abc -r viewer
DEPLOYMENT_VIEWER
 NAME     CATEGORY     ID                            APIKEY
 abc      default      ck05r3z1b07hq0d02b4tphx0o     ****************************

 Service account successfully created.
```

File: cmd/serviceaccount.go (hunk 1 above)
Comment (schnie) [reply cmd/serviceaccount.go]: yep

File: houston/queries.go [hunk 2]
```diff
@@ -106,12 +106,14 @@ var (
...
 			$category: String
 			$entityType: EntityType!
+			$role: Role!
 		) {
 			createServiceAccount(
 				entityUuid: $entityId
 				label: $label
 				category: $category
 				entityType: $entityType
+                role: $role
```
Comment (ashb) [inline houston/queries.go]: Tabs vs spaces/inconsistent indenting here.

Comment (andriisoldatenko) [reply houston/queries.go]: fixed

**Comment by andriisoldatenko** [comment]
@schnie @ashb PTAL

**Comment by andriisoldatenko** [comment]
@ashb 100% agree, i would like to refactor during astronomer/astronomer#449

**Pull request merged**